website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md

```diff
@@ -25,9 +25,9 @@ The following are the required fields for setting up a connection with a Microso
 | **Client secret** | The service principal's **client secret** (not the **client secret id**). |  
 
 
-### Active Directory password 
+### Microsoft Entra password 
 
-The following are the required fields for setting up a connection with a Microsoft Fabric using Azure AD password authentication. 
+The following are the required fields for setting up a connection with a Microsoft Fabric using Microsoft Entra password authentication. 
 
 | Field | Description |
 | --- | --- |
```

Comment:
not visible in this pr but the user and password description using 'AD'. do we want to update that to ' Microsfot Entra username / user's password'

![Screenshot 2024-03-21 at 12 36 53](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/79b19026-bcb7-4289-9e8e-442adbd33a8a)

Reply:
great catch!